### PR TITLE
[Component with its own class name as service type ] LPS-197612 Refactor Registry classes from mutiple modules

### PR DESCRIPTION
**Pull request by GabrielHenriquedev**

### CODE REVIEW

File: modules/apps/users-admin/users-admin-web/src/main/java/com/liferay/users/admin/web/internal/users/admin/management/toolbar/ServiceAccountUsersFilterContributor.java [hunk 1]
```diff
@@ -20,7 +20,10 @@
 /**
  * @author Pei-Jung Lan
  */
-@Component(service = FilterContributor.class)
+@Component(
+	property = "management.toolbar.key= " + UsersAdminManagementToolbarKeys.VIEW_SERVICE_ACCOUNTS,
```
Comment (rafaprax) [inline modules/apps/users-admin/users-admin-web/src/main/java/com/liferay/users/admin/web/internal/users/admin/management/toolbar/ServiceAccountUsersFilterContributor.java:24]: `filter.contributor.management.toolbar.key`

Comment (rafaprax) [inline modules/apps/users-admin/users-admin-web/src/main/java/com/liferay/users/admin/web/internal/users/admin/management/toolbar/ServiceAccountUsersFilterContributor.java:24]: rename to `filter.contributor.key` instead

**Review by rafaprax — CHANGES_REQUESTED** [review]

**Comment by GabrielHenriquedev** [comment]
ci:test:relevant

**Comment by Kvale1733** [comment]
@GabrielHenriquedev No regressions found. Just a semver issue.
```
 [exec] > Task :apps:frontend-taglib:frontend-taglib:baseline
     [exec]   PACKAGE_NAME                                       DELTA      CUR_VER    BASE_VER   REC_VER    WARNINGS  
     [exec] = ================================================== ========== ========== ========== ========== ==========
     [exec] * com.liferay.frontend.taglib.servlet.taglib         MAJOR      8.3.0      8.3.0      9.0.0      VERSION INCREASE REQUIRED
     [exec] 	-   class      com.liferay.frontend.taglib.servlet.taglib.ScreenNavigationRegistry
     [exec] 		-   method     <init>()
     [exec] 			-   return     void
     [exec] 		-   method     activate(org.osgi.framework.BundleContext)
     [exec] 			-   access     protected
     [exec] 			-   annotated  org.osgi.service.component.annotations.Activate
     [exec] 			-   return     void
     [exec] 		-   method     clone()
     [exec] 			-   access     protected
     [exec] 			-   return     java.lang.Object
     [exec] 		-   method     deactivate()
     [exec] 			-   access     protected
```
[Log](https://testray.liferay.com/reports/production/logs/2023-10/test-1-31/test-portal-acceptance-pullrequest(master)/9603/modules-semantic-versioning-jdk8/0/0/jenkins-console.txt.gz?authuser=0)

**Review by rafaprax — APPROVED** [review]

**Comment by GabrielHenriquedev** [comment]
ci:test:sf

**Comment by GabrielHenriquedev** [comment]
ci:test:relevant

**Comment by catarineaguiar** [comment]
@GabrielHenriquedev semver error:

```
     [exec] com.liferay.semantic.versioning.SemanticVersioningTest > testSemanticVersioning[/apps/frontend-taglib/frontend-taglib-clay] FAILED
     [exec]     java.lang.AssertionError: [Baseline Warning] Bundle Version Change Recommended: 14.0.0
     [exec]         at org.junit.Assert.fail(Assert.java:88)
     [exec]         at org.junit.Assert.assertTrue(Assert.java:41)
     [exec]         at org.junit.Assert.assertFalse(Assert.java:64)
     [exec]         at com.liferay.semantic.versioning.SemanticVersioningTest.testSemanticVersioning(SemanticVersioningTest.java:73)
     [exec]         at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
     [exec]         at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
     [exec]         at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
```
full log [here](https://testray.liferay.com/reports/production/logs/2023-10/test-1-33/test-portal-acceptance-pullrequest(master)/4492/modules-semantic-versioning-jdk8/0/0/jenkins-console.txt.gz?authuser=0)

**Comment by GabrielHenriquedev** [comment]
ci:test:sf

**Comment by GabrielHenriquedev** [comment]
ci:test:sf

**Comment by shuyangzhou** [comment]
See https://github.com/brianchandotcom/liferay-portal/pull/142324

**Pull request closed without merging**